routes/api/household.js

```diff
@@ -0,0 +1,62 @@
+const router = require("express").Router();
+const householdController = require("../../controllers/householdController");
+const mongoose = require("mongoose"); // For working with mongoose Date and ObjectId objects
+const {
+  validateOauthKey,
+  validateObjectId
+} = require("../../controllers/utils"); // Get utils.js for validation functions
+
+// All routes on this page will be appended to this root: "api/household/""
+
+router
+  .route("/")
```

Comment:
what is triggering this route?

Reply:
As per our conversation this resolves to api/household